.projen/tasks.json

```diff
@@ -0,0 +1,122 @@
+{
+  "tasks": {
+    "check-formatting": {
+      "name": "check-formatting",
+      "steps": [
+        {
+          "exec": "prettier --check src/**"
+        }
+      ]
+    },
+    "compile": {
+      "name": "compile",
+      "category": "00.build",
+      "description": "Only compile",
+      "steps": [
+        {
+          "exec": "jsii --silence-warnings=reserved-word --no-fix-peer-dependencies"
+        }
+      ]
+    },
+    "test": {
+      "name": "test",
+      "category": "10.test",
+      "description": "Run tests",
+      "steps": [
+        {
+          "exec": "rm -fr lib/"
+        },
+        {
+          "exec": "jest --passWithNoTests --all"
+        },
+        {
+          "spawn": "eslint"
+        }
+      ]
+    },
+    "build": {
+      "name": "build",
+      "category": "00.build",
+      "description": "Full release build (test+compile)",
+      "steps": [
+        {
+          "spawn": "test"
+        },
+        {
+          "spawn": "compile"
+        },
+        {
+          "spawn": "package"
+        }
+      ]
+    },
+    "bump": {
```

Comment:
How does this script work?

Reply:
If someone invokes the bump script using "yarn bump" first it must meet a condition before running. The condition is `! git log --oneline -1 | grep -q \"chore(release):\"`
I interpret this as so long the last commit in the git log history is not a `chore(release)` then the script is good to run.
The bump script executes `standard-version`, this is a [NPM](https://www.npmjs.com/package/standard-version) package "for versioning using semver and CHANGELOG generation ...". 
What does standard version do (in order)?
1. Bumps the current version in the `version.json` file. For ex. if the current version is `0.1.8` then it will be bumped to `0.1.9`.
2. Afterwards the "postbump" scripts defined in the `.versionrc.json` file will run. In this case the postbump scripts are `npx projen` & `git add .`.  `npx projen` synthesizes the project based off the `.projenrc.js` file (this also changes the package.json version # to match the version.json file so the package.json version is also bumped).
3.  A CHANGELOG.md file is  generated which basically documents what was changed in the version bumps with a nice link to compare older versions of the repo to subsequent versions.
4. Next the `version.json`, `CHANGELOG.md`, and all staged files are committed with the message `chore(release): <new version #>` for ex. `chore(release): 0.1.9`
5. A tag is created for the release in the format `v<New version #>`, for ex `v0.1.9`.
6. A message reminding the user to push your changes to the remote is printed out.

Here is the console output of when the bump script was used:
```
yarn bump
yarn run v1.22.10
$ npx projen bump
🤖 bump | condition: ! git log --oneline -1 | grep -q "chore(release):"
🤖 bump | standard-version
✔ bumping version in version.json from 0.1.8 to 0.1.9
✔ Running lifecycle script "postbump"
ℹ - execute command: "npx projen && git add ."
🤖 Synthesizing project...
🤖 yarn install --check-files
[1/4] Resolving packages...
success Already up-to-date.
🤖 Synthesis complete

✔ created CHANGELOG.md
✔ outputting changes to CHANGELOG.md
✔ committing version.json and CHANGELOG.md and all staged files
✔ tagging release v0.1.9
ℹ Run `git push --follow-tags origin AR/ProjenDatadogCDKConstruct` to publish
✨  Done in 6.25s.
```